lib/docs/filters/nuxtjs/container.rb

```diff
@@ -0,0 +1,9 @@
+module Docs
+  class Nuxtjs
+    class ContainerFilter < Filter
+      def call
+          at_css '.docus-content'
```

Comment:
Simply use `options[:container] = '.docus-content'` in lib/docs/scrapers/nuxtjs.rb

Reply:
I believe I tried that but it didn't end up following links and thus didn't do a full scrape. But I guess will have to tackle the license issue first 😅